RecoParticleFlow/PFClusterProducer/interface/PFECALHashNavigator.h

```diff
@@ -260,7 +260,7 @@ bool stdsimplemove(DetId& cell,
 	
 	// yes we are
 	const EcalBarrelGeometry::OrderedListOfEEDetId& 
-	  ol( * barrelGeom.getClosestEndcapCells( ebDetId ) ) ;
+	  ol( * ((EcalBarrelGeometry*)(&barrelGeom))->getClosestEndcapCells( ebDetId ) ) ;
```

Comment:
Remove cast. If the type conversion is needed use `dynamic_cast<EcalBarrelGeometry const*>(&barrelGeom)`